docs/how-to-guides/admin/deploying-renku.rst

```diff
@@ -209,13 +236,13 @@ GitLab deployed as part of Renku
 
 If your RenkuLab deployment includes GitLab you need to follow some additional steps to configure an admin user on GitLab.
 
-#. turn off automatic redirection to GitLab by redeploying with the value ``gitlab.oauth.autoSignIn: false``
-#. log in as the root user using the password from ``gitlab.password``
-#. modify any users you want to modify (e.g. to make them admin)
-#. turn the ``autoSignIn`` setting back on
+#. Turn off automatic redirection to GitLab by setting the value ``gitlab.oauth.autoSignIn=false`` and redeploy with ``helm``.
+#. Log in as the root user using the password from ``gitlab.password``.
+#. Modify any users you want to modify (e.g. to make them admin).
+#. Turn the ``autoSignIn`` setting back on and redeploy.
```

Comment:
We have to redeploy renku just because admin users were added to gitlab? Why is this the case?

Reply:
This is just to log in as the root user and make a normal user an admin. There is no other way to log in as the root gitlab user because the root user doesn't exist as a renku user.